packages/flutter/lib/src/widgets/text.dart

```diff
@@ -244,7 +244,7 @@ class Text extends StatelessWidget {
        textSpan = null,
        super(key: key);
 
-  /// Creates a text widget with a [TextSpan].
+  /// Creates a text widget with a [InlineSpan].
   ///
   /// The [textSpan] parameter must not be null.
   const Text.rich(
```

Comment:
This parameter is called "textSpan" currently even though it now is of type `InlineSpan`. If we leave it as it is, it will not break anyone, but since this is a positional parameter, we could also rename it to 'inlineSpan' without breaking. The member 'textSpan' variable could not make this change without breaking though. What is the best course of action here?

Reply:
let's just leave it, but when you land the patch, update https://github.com/flutter/flutter/issues/24722